selenium_tests/page.py

```diff
@@ -0,0 +1,32 @@
+""""""
+
+from selenium.webdriver.common.by import By
+
+
+class BasePage(object):
+    def __init__(self, browser):
+        self.browser = browser
+
+
+class LoginPage(BasePage):
```

Comment:
starting in on an implementation of this pattern: http://selenium-python.readthedocs.io/page-objects.html